content/en/profiler/enabling/go.md

```diff
@@ -22,7 +22,9 @@ The profiler is shipped within Datadog tracing libraries. If you are already usi
 
 ## Requirements
 
-The Datadog Profiler requires Go 1.12+.
+The Datadog Profiler requires Go 1.12+. For Endpoints profiling.
```

Comment:
```suggestion
The Datadog Profiler requires Go 1.12+ for Endpoints profiling.
```

Reply:
Ah, this is a stray paste. Endpoints and CH both require 1.18+